manifests/repo.pp

```diff
@@ -1,49 +1,53 @@
-# private class
-class sensu::repo {
+# @summary Private class to manage sensu repository resources
+# @api private
+#
+class sensu::repo (
+  $manage_repo = $sensu::manage_repo
+) inherits sensu {
```

Comment:
I'd recommend not inheriting sensu but instead doing `include ::sensu` and then pulling in value of `manage_repo` from that class so the value has to be changed in `sensu` class.

Reply:
done